src/components/SpecialResolution/SpecialResolutionSummary.vue

```diff
@@ -0,0 +1,333 @@
+              The alteration for this business will be effective as of:<br>
+              <strong>{{effectiveDateTimeString}}</strong>
+            </v-card>
+          </v-col>
+        </v-row>
+      </v-container>
+    </div>
+  </v-card>
+</template>
+
+<script lang="ts">
+// this is a placceholder copied from AlterationSummary, Will add component when working on this page
+import { Component, Emit, Mixins, Prop } from 'vue-property-decorator'
+import { Action, Getter } from 'vuex-class'
+import { ActionBindingIF, FlagsReviewCertifyIF, FeesIF, ResolutionsIF } from '@/interfaces/'
+import { DateMixin, SharedMixin, FilingTemplateMixin, PayApiMixin } from '@/mixins/'
+import { EffectiveDateTime, NameTranslation, ShareStructures } from '@/components/common/'
+import { ResolutionDates } from '@/components/Alteration/'
+
+@Component({
+  components: {
+    EffectiveDateTime,
+    NameTranslation,
+    ResolutionDates,
+    ShareStructures
+  }
+})
+export default class SpecialResolutionSummary extends Mixins(
+  DateMixin,
+  SharedMixin,
+  FilingTemplateMixin,
+  PayApiMixin
+) {
+  // Global getters
+  @Getter getBusinessNumber!: string
+  @Getter getOriginalResolutions!: ResolutionsIF[]
+  @Getter getCurrentFees!: FeesIF
+  @Getter isBusySaving!: boolean
+  @Getter getFeePrices!: FeesIF
+  @Getter getFlagsReviewCertify!: FlagsReviewCertifyIF
+  @Getter haveNewResolutionDates!: boolean
+  @Getter getCurrentJsDate!: Date
+
+  // Global actions
+  @Action setSummaryMode!: ActionBindingIF
+  @Action setEffectiveDateValid!: ActionBindingIF
+
+  /** Whether to perform validation. */
+  @Prop() readonly validate: boolean
+
+  get isFutureEffective (): boolean {
+    return this.getEffectiveDateTime.isFutureEffective
+  }
+
+  get isEffectiveDateTimeValid (): boolean {
+    return this.getFlagsReviewCertify.isValidEffectiveDate
+  }
+
+  get effectiveDateTimeString (): string {
+    const date = new Date(this.getEffectiveDateTime.dateTimeString)
+    return this.dateToPacificDateTime(date)
+  }
+
+  /** The company name (from NR, or incorporation number). */
+  get companyName (): string {
+    if (this.getNameRequestLegalName) return this.getNameRequestLegalName
+
+    return `${this.getBusinessNumber || '[Incorporation Number]'} B.C. Ltd.`
+  }
+
+  get originalLegalType (): string {
+    return this.getEntitySnapshot?.businessInfo?.legalType
+  }
+
+  /** True if invalid class should be set for Alteration Date-Time container. */
+  get alterationDateTimeInvalid (): boolean {
+    return (this.validate && !this.getFlagsReviewCertify.isValidEffectiveDate)
+  }
+
+  // sum of alteration fees
+  get alterationFees (): string {
+    if (this.getCurrentFees.filingFees !== null && this.getCurrentFees.futureEffectiveFees !== null) {
+      return `($${(this.getCurrentFees.filingFees + this.getCurrentFees.futureEffectiveFees).toFixed(2)} Fee)`
+    }
+    return ''
+  }
+
+  get futureEffectiveFeePrice (): string {
```

Comment:
You don't need this if future effective Special Resolutions aren't supported.

Maybe you don't need alterationDateTimeInvalid() or some template components either.